Dockerfile.stream8

```diff
@@ -15,5 +15,21 @@ RUN dnf install -y \
         gzip \
         make \
         rpm-build \
-        tar
+        tar \
+        dnf-utils
+
 COPY containerroot /
+
+WORKDIR /tmp/src-rpm
+# Download source rpms and install it's dependencies
+RUN reposync --repoid=copr:copr.fedorainfracloud.org:ovirt:ovirt-master-snapshot --source --newest-only \
+    && \
+    find . -iname '*.src.rpm' -exec mv {} . \; \
+    && \
+    dnf builddep -y -D 'enable_autotools 1' *.src.rpm \
```

Comment:
enable_autotools (for vdms) not needed since it's pulled in by other packages. I'd remove it here for consistency, even if technically needed for vdsm alone.

Reply:
Done